### DISCUSSION TITLE
Show page returns 404

### DISCUSSION THREAD

**Discussion by abdellahrk** [manuxi/SuluNewsBundle#8] [Q&A]
Hello, how is the single blog/article shown? Should it be overwritten?
Tring to access a single news with the `routePath()` shows 404 with error like `No route found for GET https:://myhost/news/news-item`.
The hack is for me to append the query string like the id to the url and fetch with a custom controller and it makes me reveal the id of the news item in the url.
Thank you.

**Comment by manuxi** [comment]
This should work. I tested it in several installations. 
So do you see the preview? What route path did you enter in the news item? What is displayed there atm? Can you share a screenshot of the news item? Do you have rules in htaccess or similar? Did you try a fresh install?

**Comment by abdellahrk** [comment]
Hi @manuxi, thank you manifold.
It works espcially this https://github.com/manuxi/SuluNewsBundle/issues/5#issuecomment-1790607708 helped a lot.
